tests/test_basic.py

```diff
@@ -169,3 +170,18 @@ def test_invalid_task_to_compute(self):
             s = msg.serialize()
             msg2 = message.Message.deserialize(s, decrypt_func=None)
             self.assertIs(msg2, None)
+
+
+class ComputeTaskDefTestCase(unittest.TestCase):
+    def test_type(self):
+        ctd = message.ComputeTaskDef()
+        ctd['src_code'] = ("Nie mamy w Polsce monarchy,"
```

Comment:
I cant review this part, should it be in polish?

Reply:
:) It's just a poem by Tadeusz Boy-Żeleński. If you feel uncomfortable with this I can remove it. It does not affect the test.

Reply:
Cool! no need, maybe a comment above it that it is just test data / lorem ipum would be enough :)